src/corppa/poetry_detection/annotation/command_recipes.py

```diff
@@ -0,0 +1,87 @@
+from collections import Counter, defaultdict
+
+from prodigy.components.db import connect
+from prodigy.core import Arg, recipe
+from prodigy.errors import RecipeError
+from prodigy.util import SESSION_ID_ATTR, msg
+
+
+@recipe(
+    "page-stats",
+    dataset=Arg(help="Prodigy dataset ID"),
+)
+def ppa_stats(dataset: str) -> None:
+    # Load examples
+    DB = connect()
+    if dataset not in DB:
+        raise RecipeError(f"Can't find dataset '{dataset}' in database")
+    examples = DB.get_dataset_examples(dataset)
+    n_examples = len(examples)
+    msg.good(f"Loaded {n_examples} annotations from {dataset} dataset")
+
+    # Get stats
+    examples_by_page = Counter()
+    examples_by_session = defaultdict(list)
+    for ex in examples:
+        # Skip examples w/ answer or (page) id
+        if "answer" not in ex and "id" not in ex:
+            # Ignore "unanswered" examples
+            continue
+        page_id = ex["id"]
+        examples_by_page[page_id] += 1
+        session_id = ex[SESSION_ID_ATTR]
+        examples_by_session[session_id].append(page_id)
+    # Get frequencies of page-level annotation counts
+    count_freqs = Counter()
+    total = 0
+    for count in examples_by_page.values():
+        count_freqs[count] += 1
+        total += count
```

Comment:
What you're doing here wasn't immediately obvious based on the variable names - am I understanding correctly that you're counting the number of examples/pages that have the same number of annotations, so you can report something like 100 examples have 1 annotation each, 50 have 2 annotation each, etc? 

You can let `Counter` do the aggregation for you by using `Counter(examples_by_page.values())` .

I think it would be more readable to tally this way:
```suggestion
    # Get frequencies of page-level annotation counts
    count_freqs = Counter(examples_by_page.values())
    total = Sum(examples_by_page.values())
```

Reply:
These are the count frequencies, the frequency at which each page image has been annotated so far (like you described)

Good point on the `Counter` / `sum` usage. Not sure why I missed that.